src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -212,7 +212,7 @@ private void leaveContainerNode(DetailAST ast) {
      */
     private void visitSlist(DetailAST ast) {
         final DetailAST contextAST = context.getAST();
-        DetailAST parent = ast.getParent();
+        DetailAST parent = ast;
```

Comment:
Similar reasoning to https://github.com/checkstyle/checkstyle/pull/12588#issue-1520971449 . `isContainerNode` checks for specific types, and `SLIST` isn't one of them, so it just adds a bit more processing time to figure out it has to go up one anyways.